Emby.Server.Implementations/ScheduledTasks/Tasks/AudioNormalizationTask.cs

```diff
@@ -0,0 +1,195 @@
+public partial class AudioNormalizationTask : IScheduledTask
+{
+    private readonly IItemRepository _itemRepository;
+    private readonly ILibraryManager _libraryManager;
+    private readonly IMediaEncoder _mediaEncoder;
+    private readonly IConfigurationManager _configurationManager;
+    private readonly ILocalizationManager _localization;
+    private readonly ILogger<AudioNormalizationTask> _logger;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AudioNormalizationTask"/> class.
+    /// </summary>
+    /// <param name="itemRepository">Instance of the <see cref="IItemRepository"/> interface.</param>
+    /// <param name="libraryManager">Instance of the <see cref="ILibraryManager"/> interface.</param>
+    /// <param name="mediaEncoder">Instance of the <see cref="IMediaEncoder"/> interface.</param>
+    /// <param name="configurationManager">Instance of the <see cref="IConfigurationManager"/> interface.</param>
+    /// <param name="localizationManager">Instance of the <see cref="ILocalizationManager"/> interface.</param>
+    /// <param name="logger">Instance of the <see cref="ILogger{AudioNormalizationTask}"/> interface.</param>
+    public AudioNormalizationTask(
+        IItemRepository itemRepository,
+        ILibraryManager libraryManager,
+        IMediaEncoder mediaEncoder,
+        IConfigurationManager configurationManager,
+        ILocalizationManager localizationManager,
+        ILogger<AudioNormalizationTask> logger)
+    {
+        _itemRepository = itemRepository;
+        _libraryManager = libraryManager;
+        _mediaEncoder = mediaEncoder;
+        _configurationManager = configurationManager;
+        _localization = localizationManager;
+        _logger = logger;
+    }
+
+    /// <inheritdoc />
+    public string Name => _localization.GetLocalizedString("TaskAudioNormalization");
+
+    /// <inheritdoc />
+    public string Description => _localization.GetLocalizedString("TaskAudioNormalizationDescription");
+
+    /// <inheritdoc />
+    public string Category => _localization.GetLocalizedString("TasksLibraryCategory");
+
+    /// <inheritdoc />
+    public string Key => "AudioNormalization";
+
+    [GeneratedRegex(@"I:\s+(.*?)\s+LUFS")]
+    private static partial Regex LUFSRegex();
+
+    /// <inheritdoc />
+    public async Task ExecuteAsync(IProgress<double> progress, CancellationToken cancellationToken)
+    {
+        foreach (var library in _libraryManager.RootFolder.Children)
+        {
+            var libraryOptions = _libraryManager.GetLibraryOptions(library);
+            if (!libraryOptions.EnableLUFSScan)
+            {
+                continue;
+            }
+
+            // Album gain
+            var albums = _libraryManager.GetItemList(new InternalItemsQuery
+            {
+                IncludeItemTypes = [BaseItemKind.MusicAlbum],
+                Parent = library,
+                Recursive = true
+            });
+
+            foreach (var a in albums)
```

Comment:
Should add a `try..catch` so we don't stop processing on first (or any) failure

Reply:
There is a try catch inside of `CalculateLUFSAsync` where I'd expect these failures to take place.
Seems unlikely anything would go wrong here, and it it does (like failing to create the concat file) it would likely happen to the other items as well